src/encoding/src/tests/sol_abi.cairo

```diff
@@ -257,9 +260,8 @@ fn decode_test() {
     assert_eq!(offset, 256, "Offset after decode ByteArray failed");
 
     let decoded: bytes31 = encoded.decode(ref offset);
-    assert!(
-        decoded == bytes31_const::<0xa0aaab00000000000000000000000000ac>(), "Decode byte31 failed"
-    );
+    let bytes_31: bytes31 = 0xa0aaab00000000000000000000000000ac.try_into().unwrap();
+    assert!(decoded == bytes_31, "Decode byte31 failed");
```

Comment:
```suggestion
    assert_eq!(decoded, bytes_31, "Decode byte31 failed");
```

Reply:
Did where I could (bigass regex, and reverted the file where it was causing issues), because some types issues, would re-do a pass in another PR to avoid having too much conflicts